### PR TITLE
ui enhancements for earnings

### PR DESCRIPTION
**Pull request by jvmsangkal**

### CODE REVIEW

**Comment by ravenjohn** [comment]
<a href="http://4380e460.ngrok.com/anyTV/music-dashboard/pull/378" target="_blank">ReviewNinja

**Comment by sibayanjasper** [comment]
:AllenApproves:

File: app/scripts/actions/earnings_actions.js [hunk 1]
```diff
@@ -8,7 +8,9 @@ var Reflux = require('reflux'),
...
         'loadPerUser',
-        'loadPerMonth'
+        'loadPerMonth',
+        'loadMonths',
+        'changeMonth'
     ]);
 
-module.exports = EarninigsActions;
\ No newline at end of file
+module.exports = EarninigsActions;
```
Comment (sibayanjasper) [inline app/scripts/actions/earnings_actions.js]: I think it should be `EarningsActions`

Comment (ravenjohn) [inline app/scripts/actions/earnings_actions.js]: it should really be `EarningsActions`. get your spelling right @jvmsangkal . !fix

Comment (jvmsangkal) [inline app/scripts/actions/earnings_actions.js]: !fixed

**Comment by rycastaneda** [comment]
:star: :AllenApproves:

**Comment by sibayanjasper** [comment]
:AllenApproves:

**Comment by sibayanjasper** [comment]
:+1:

**Comment by pprmint** [comment]
:+1:

**Comment by sibayanjasper** [comment]
:+1:

**Pull request merged**